server/src/test/java/com/google/coffeehouse/common/PersonTest.java

```diff
@@ -36,69 +41,105 @@
   private static final String ALT_PRONOUNS = "New Pronouns";
   private static final String IDENTIFICATION_STRING = "predetermined-identification-string";
   private Person.Builder personBuilder;
+  private Map personInfo;
   
   @Mock private IdentifierGenerator idGen;
 
   @Before
   public void setUp() {
-    idGen = Mockito.mock(IdentifierGenerator.class);
-    Mockito.when(idGen.generateId()).thenReturn(IDENTIFICATION_STRING);
+    personInfo = new HashMap<String, String>();
+
+    idGen = mock(IdentifierGenerator.class);
+    when(idGen.generateId()).thenReturn(IDENTIFICATION_STRING);
 
     personBuilder = Person.newBuilder(EMAIL, NICKNAME);
   }
 
   @Test
-  public void getExistingNickname() {
+  public void getNickname_exists() {
     Person p = personBuilder.build();
     Assert.assertEquals(NICKNAME, p.getNickname());
```

Comment:
optional nit: I recommend you statically import assertEquals (and its kin) too, so you don't have to do Assert. every time.

You can import the whole group as `import static org.junit.Assert.*`

Reply:
That makes sense, definitely lowers line length and increases readability. I have implemented the static import on these testing files.